registry/proxy/proxyblobstore.go

```diff
@@ -67,6 +68,24 @@ func (pbs *proxyBlobStore) copyContent(ctx context.Context, dgst digest.Digest,
 	return desc, nil
 }
 
+func (pbs *proxyBlobStore) doServeBlobResponse(ctx context.Context, dgst digest.Digest, writer http.ResponseWriter, reqMethod string) error {
```

Comment:
Let's have a note here describing why this exists so that, in the future, we know why there is a seemingly useless method.